Kickstarter-iOS/Features/ReportProject/ReportProjectFormView.swift

```diff
@@ -8,45 +8,54 @@ enum ReportFormFocusField {
 
 @available(iOS 15.0, *)
 struct ReportProjectFormView: View {
-  let projectID: String
```

Comment:
I moved these properties into the view model for two reasons:
1) The view model needs `projectId` and `projectFlaggingKind` to submit the report project mutation
2) Just in general, it seems more MVVM-y to me to have the view model own this state, instead of the view

Reply:
Agreed. I mentioned in a comment below that we could consider injecting the VM with these properties from the outset instead of having the view be something of a middleman here.